miri

```diff
@@ -211,6 +214,10 @@ fmt)
     find "$MIRIDIR" -not \( -name target -prune \) -name '*.rs' \
         | xargs rustfmt +$TOOLCHAIN --edition=2021 --config-path "$MIRIDIR/rustfmt.toml" "$@"
     ;;
+cargo)
+    find_sysroot
```

Comment:
What is this good for? The called cargo-miri should be doing sysroot stuff itself, shouldn't it?

Reply:
yea, that's indeed not needed, just copy pasted ^^